src/strategies/frankendao-votingpower/index.ts

```diff
@@ -0,0 +1,34 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'NFT-Utils';
+export const version = '1.0.0';
+
+const abi = [
+  'function getVotes(address _account) public view returns (uint)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.address, 'getVotes', [address])
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, votingPower]) => [
+      address,
+      parseFloat(formatUnits(votingPower, 0))
+    ])
+  );
```

Comment:
I feel like the `contract-call` strategy should work perfectly in your case :) Can try it?

Reply:
Or also https://snapshot.org/#/strategy/erc20-votes

Reply:
Good callout, let me try these first.